webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue

```diff
@@ -54,7 +54,7 @@
             <v-img :src="item.profile.avatarUrl"></v-img>
           </v-avatar>
           <span class="text-truncate">
-            {{ item.profile.fullName }}
+            {{ item.profile.external !== null && item.profile.external ? item.profile.fullName.concat(' (').concat($t('userAvatar.external.label')).concat(')') : item.profile.fullName }}
```

Comment:
Is it possible to compute it ?

Reply:
It's not possible to use compute, in this case, it's related to item the external and not to items witch is declared.